install_config/storage_examples/storage_classes_dynamic_provisioning.adoc

```diff
@@ -0,0 +1,322 @@
+====
+[source,yaml]
+----
+kind: StorageClass
+apiVersion: storage.k8s.io/v1beta1
+metadata:
+  name: default <1>
+  annotations:
+    storageclass.beta.kubernetes.io/is-default-class: "true" <2>
+provisioner: kubernetes.io/gce-pd
+parameters:
+  type: pd-standard
+  zone: us-east1-d
+----
+<1>  Name of the StorageClass.
+<2>  Annotation that marks this StorageClass as the default class.  You must use the "true" quoted in this version of the api. If annotation is not supplied, it by default implies that this is not the _default_ StorageClass.
+====
+
+As a cluster-admin or storage-admin save the definition to a .yaml file, (i.e. default-gce.yaml), and then create the StorageClasses.
+
+====
+----
+
+# oc create -f default-gce.yaml 
+storageclass "default" created
+
+# oc get storageclass
+NAME       TYPE
+default    kubernetes.io/gce-pd   
+fast       kubernetes.io/gce-pd   
+slow       kubernetes.io/gce-pd   
+
+----
+====
+
+As a user, create a new claim without any StorageClass annotation, by creating the claim yaml definition and saving it to a file (i.e. default-pvc.yaml), and then execute it and view if the claim was bound.
+
+
+.Claim for Fast Storage Spec Definition
+====
+[source,yaml]
+----
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+ name: pvc-engineering2
+spec:
+ accessModes:
+  - ReadWriteMany
+ resources:
+   requests:
+     storage: 5Gi
+----
+====
+
+====
+----
+
+# oc create -f default-pvc.yaml 
+persistentvolumeclaim "pvc-engineering2" created
+                                                                   3s
+[root@screeley-sc1 gce]# oc get pvc
+NAME               STATUS    VOLUME                                     CAPACITY   ACCESSMODES   AGE
+pvc-engineering    Bound     pvc-e9b4fef7-8bf7-11e6-9962-42010af00004   10Gi       RWX           41m
+pvc-engineering2   Bound     pvc-a9f70544-8bfd-11e6-9962-42010af00004   5Gi        RWX           7s
+
+----
+====
+
+
+As a cluster-admin or storage-admin view the Persistent Volumes that are defined so far
+
+====
+----
+
+# oc get pv
+NAME                                       CAPACITY   ACCESSMODES   RECLAIMPOLICY   STATUS    CLAIM                     REASON    AGE
+pvc-a9f70544-8bfd-11e6-9962-42010af00004   5Gi        RWX           Delete          Bound     rh-eng/pvc-engineering2             5m <1>
+pvc-ba4612ce-8b4d-11e6-9962-42010af00004   5Gi        RWO           Delete          Bound     mytest/gce-dyn-claim1               21h
+pvc-e9b4fef7-8bf7-11e6-9962-42010af00004   10Gi       RWX           Delete          Bound     rh-eng/pvc-engineering              46m <2>
+
+----
+<1> This PV was bound to our _default_ dynamic volume from the _default_ StorageClass.
+<2> This PV was bound to our first PVC from example 1 with our _fast_ StorageClass.
+====
+
+Create a manual disk in the GCE console (not dynamically provisioned) and a Persistent Volume that connects to the new GCE disk.
+
+Now view the PVs again, notice we have a pv-manual-gce volume Available.
```

Comment:
This doesn't show specifically how to create the PV but then makes strick assumptions about how it was created...

While I don't think the 'create a manual disk a PV' need to be inline a link to a doc to show your exact steps would be good/helpful